.changeset/bright-dog-sad.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ledgerhq/live-common": minor
```

Comment:
Hi,

Could you change `minor` to `patch`?

Reply:
hello, sure!

Reply:
updated